code/game/objects/structures/girders.dm

```diff
@@ -84,7 +84,7 @@
 	if(istype(W, /obj/item/weapon/melee/twohanded/breacher))
 		if(user.action_busy)
 			return
-		if(!HAS_TRAIT(user, TRAIT_SUPER_STRONG))
+		if(!(HAS_TRAIT(user, TRAIT_SUPER_STRONG)) && istype(W, /obj/item/weapon/melee/twohanded/breacher/synth))
```

Comment:
I believe you can just remove this check because you can't attack with the synth hammer without being super strong. Unsure tho so test on local